index.html 

```diff
@@ -1,47 +1,48 @@
 <!DOCTYPE html>
-<head>
-	<title>Bubble Tea site</title>
-</head>
-<body>
-	<div class="topnav">
-		<img src="" alt="logo" style="width:600px; height:400px"/>
-		<a class="active" href=#home">Home</a>
-		<a href=#news">News</a>
-		<a href=#about">About</a>
-		<a href=#contact”>Contact</a>
-	</div
-	<div>
-		<img src="" alt="description of image">
-	</div>
-	<div> 
-		<p>text</p>
-	</div>
-    <div>
-		<p>text</p>
-	</div>
-	<div>
-		<img src="" alt="description of image"/>
-	</div>
-	<div>
-		<p>Text</p>
-		<button>More</button>
-	</div>
-	<table>
-		<tr>
-			<th>Text</th>
-			<th>Text</th>
-			<th>Text</th>
-		</tr>	
-		<tr>
-			<td>text</td>
-			<td>text</td>
-			<td>text</td>
-		</tr>
-	</table>
-	<footer>
-		<a href=#copyright">copyright 2022</a>
-		<a href=#terms">Terms of Conditions</a>
-		<a href=#contact">Contact Us</a>
-	</footer>
- </body>
+<html>
```

Comment:
Extra `<html>` here.
When you have `<!DOCTYPE html>` above on line 1, you don't need `<html>`, as they serve the same purpose.